### PR TITLE
Add check for value 0 + correct integration of pointToLayer factory method + permit custom header

### PR DESCRIPTION
**Pull request by wouldgo**
Hi, 

today using your library to achieve a WFST integration I spot that if a value is provided as 0 the save process skips those values bringing my update in error.

I modified the script to achieve this. The check for a falsy value brought also 0 to be treated as ''.

Hope that this can be useful,
Dario.

### CODE REVIEW

**Comment by wouldgo** [comment]
I spot also an issue on using the `pointToLayer` factory method. The `L.GeoJSON.geometryToLayer` wants the options object.

**Comment by wouldgo** [comment]
For a use case I have to send also custom headers to the map server.
If you prefer a splitted series of PRs for this contributions tell me.

Best regards,

Dario

File: src/Format/Format.GeoJSON.js [hunk 1]
```diff
@@ -28,6 +28,6 @@ L.Format.GeoJSON = L.Format.Base.extend({
   },
 
   generateLayer: function (feature) {
-    return L.GeoJSON.geometryToLayer(feature, this.options.pointToLayer || null, this.options.coordsToLatLng || null, null);
+    return L.GeoJSON.geometryToLayer(feature, this.options || null, this.options.coordsToLatLng || null, null);
```
Comment (kuzkok) [inline src/Format/Format.GeoJSON.js:31]: [geometryToLayer ](http://leafletjs.com/reference-1.0.2.html#geojson-geometrytolayer) have 2 arguments, I don't remeber for which purpose here were be 4 :), could you also remove unnecessary args?

**Comment by kuzkok** [comment]
For future PRs, please, split it by features.

File: src/Format/Format.GeoJSON.js (hunk 1 above)
Comment (wouldgo) [reply src/Format/Format.GeoJSON.js:31]: can be done, I left the other agrs due to I didn't get why there were 4

**Pull request merged**